scripts/schemas/board-schema.yml

```
@@ -30,6 +30,10 @@ schema;board-schema:
       required: true
       type: str
       desc: Name of the board
+    description:
+      required: false
+      type: str
+      desc: description or full name of the board
```

We should be more restrictive. "description **or** full name" can lead to a lot of inconsistencies, with some folks going for "FRDM-K64F" (a "full name") and others for "Ultra-low-cost development platform for Kinetis® K64, K63, and K24 MCUs" (a "description").

Since what we really want (I think?) is the `commercial_name` of the board, maybe actually call the property that (and update the desc accordingly). Or, given that some boards are not really commercialized, maybe settle for:

```suggestion
    full_name:
      required: false
      type: str
      desc: Full name of the board. Typically set to the commercial name of the board.  
```